doc/quickstart/configure.rst

```diff
@@ -370,10 +370,9 @@ These four items here are named people, references and projects listed in the
 Extra Facets
 ============
 
-Sometimes it is useful to provide extra information for the loading of data,
-particularly in the case of native model data, or observational or other data,
-that generally follows the established standards, but is not part of the big
-supported projects like CMIP, CORDEX, obs4MIPs.
+Sometimes it is useful to automatically add extra key-value pairs to variables
+or datasets in the recipe.
+These key-value pairs are called facets.
```

Comment:
Perhaps we can find a blend of the old and the new description? I think the opening sentence here should make it immediately clear what this is and why it's useful. I think this is now missing a bit.